include/irods/private/storage_tiering/storage_tiering.hpp

```diff
@@ -1,12 +1,16 @@
 #ifndef IRODS_CAPABILITY_STORAGE_TIERING_HPP
 #define IRODS_CAPABILITY_STORAGE_TIERING_HPP
 
-#include <list>
+#include "irods/private/storage_tiering/configuration.hpp"
+#include <irods/rcMisc.h>
```

Comment:
Let's add an empty line between these headers so it's clear that one is provided by the tiering implementation and the other is from an external source.

Reply:
Done